### DISCUSSION TITLE
open on workspace by id

### DISCUSSION THREAD

**Discussion by aquifolly** [YaLTeR/niri#656] [Ideas]
allow the `open-on-workspace` rule to specify workspace by id

**Comment by YaLTeR** [comment]
This does not really make sense as workspace IDs are not stable or something you should rely on being a particular value. I am reserving the right to change how they are computed (so they are not guaranteed to keep counting from 1).

**Reply by YaLTeR** [reply]
What do you want to do with it?

**Reply by aquifolly** [reply]
when niri starts i'd like to open apps on the first two workspaces, but i'd rather not have to use named workspaces for that

**Reply by YaLTeR** [reply]
Why not?

**Reply by aquifolly** [reply]
now that I think about it, my current workflow (with named workspaces) works fine
